forge/src/main/java/org/openjdk/skara/forge/github/GitHubPullRequest.java

```diff
@@ -703,6 +703,20 @@ public void makeNotDraft() {
             .execute();
     }
 
+    @Override
+    public Optional<ZonedDateTime> lastMarkedAsDraftTime() {
+        var lastMarkedAsDraftTime = request.get("issues/" + json.get("number").toString() + "/timeline")                .execute().stream()
+                .map(JSONValue::asObject)
+                .filter(obj -> obj.contains("event"))
+                .filter(obj -> obj.get("event").asString().equals("convert_to_draft"))
+                .map(obj -> ZonedDateTime.parse(obj.get("created_at").asString()))
+                .reduce((a, b) -> a.isBefore(b) ? b : a);
```

Comment:
I think this could be expressed as `max()`.

Reply:
Fixed.